src/js/components/Grommet/__tests__/Grommet-test.tsx

```diff
@@ -90,32 +92,28 @@ describe('Grommet', () => {
       <Grommet
         messages={{
           messages: {
-            test: {
-              label: 'My Label',
+            button: {
+              busy: 'My Label',
             },
           },
         }}
       >
         <MessageContext.Consumer>
-          {({ format }) => format({ id: 'test.label' })}
+          {({ format }: MessageContextValue) => format({ id: 'button.busy' })}
         </MessageContext.Consumer>
       </Grommet>,
     );
     expect(container.firstChild).toMatchSnapshot();
   });
 
   test('message format function', () => {
-    const messages = {
+    const messages: Record<string, string> = {
       'test.label': 'My Label',
     };
     const { container } = render(
-      <Grommet full background="#0000ff">
-        Grommet App
-      </Grommet>,
-
-      <Grommet messages={{ format: (opts) => messages[opts.id] }}>
+      <Grommet messages={{ format: (opts) => messages[opts.id || ''] }}>
```

Comment:
```suggestion
      <Grommet messages={{ format: (opts) => messages[opts.id] }}>
```

since both results in undefined

```js
messages[opts.id] // undefined
messages[opts.id || ''] // undefined  
```